pytorch_lightning/accelerators/cpu_backend.py

```diff
@@ -13,12 +13,13 @@
 # limitations under the License.
 
 from pytorch_lightning.utilities.exceptions import MisconfigurationException
+from pytorch_lightning.accelerators.base_backend import Accelerator
 
 
-class CPUBackend(object):
+class CPUBackend(Accelerator):
```

Comment:
lets call it `Lightning Accelerator`

Reply:
patience... this is the last thing on the refactors...